src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/CaseDetails.java

```diff
@@ -47,10 +47,6 @@ public String getJurisdiction() {
 
     public State getState() {
 
-        if (state == null) {
```

Comment:
Why are you removing this check?

Reply:
I have introduced a preparer class (FeePaymentPreparer) that triggers on start_appeal and at this stage CaseDetails does not have a createdDate or Id.